### PR TITLE
Fix/lint

### PR DESCRIPTION
**Pull request by heynemann**
Fixing all pylint warning in the codebase.

### CODE REVIEW

**Comment by heynemann** [comment]
image

File: thumbor/loaders/http_loader.py [hunk 1]
```diff
@@ -72,23 +74,22 @@ def validate(context, url, normalize_url_func=_normalize_url):
...
+            f"original_image.fetch.{code}.{netloc}",
             (finish - req_start).total_seconds() * 1000,
         )
         context.metrics.incr(
-            "original_image.fetch.{0}.{1}".format(response.code, res.netloc.replace(".", "_"))
+            f"original_image.fetch.{code}.{netloc}",
         )
 
     result = LoaderResult()
     context.metrics.incr("original_image.status." + str(response.code))
```
Comment (cristiandean) [inline thumbor/loaders/http_loader.py:91]: Replace `response.code` with previously declared `code`

File: thumbor/engines/extensions/exif_orientation_editor.py [hunk 2]
```diff
@@ -17,7 +17,10 @@ class ExifOrientationEditor:
 
     def __init__(self, data):
         if data[:6] != EXIF_HEADER:
-            raise SyntaxError("not a TIFF file (header %r not valid)" % data[:4])
+            raise SyntaxError(
+                "not a TIFF file (header %r not valid)"  # pylint: disable=consider-using-f-string
```
Comment (cristiandean) [inline thumbor/engines/extensions/exif_orientation_editor.py:21]: Is there any reason not to use _f string_ here?

File: thumbor/engines/extensions/exif_orientation_editor.py [hunk 3]
```diff
@@ -29,7 +32,10 @@ def _read_header(self):
         header = self.exif_buffer.read(8)
 
         if header[:4] not in PREFIXES:
-            raise SyntaxError("not a TIFF file (header %r not valid)" % header)
+            raise SyntaxError(
+                "not a TIFF file (header %r not valid)"  # pylint: disable=consider-using-f-string
```
Comment (cristiandean) [inline thumbor/engines/extensions/exif_orientation_editor.py:36]: And here

**Review by cristiandean — COMMENTED** [review]
Great job! 🙂

File: setup.py [hunk 4]
```diff
@@ -139,9 +139,9 @@ def run_setup(extension_modules=None):
 
 try:
     run_setup(gather_filter_extensions())
-except SystemExit as exit:
-    print("\n\n%s" % ("*" * 66))
-    logging.exception(exit)
-    print("\n\n%s" % ("*" * 66))
+except SystemExit as exit_error:
+    print("\n\n{'*' * 66}")
```
[the remark this answers is not in the document]
Comment (heynemann) [reply setup.py]: Great catch!

File: thumbor/loaders/http_loader.py (hunk 1 above)
Comment (heynemann) [reply thumbor/loaders/http_loader.py]: Not really. Fixed! 👍

File: thumbor/engines/extensions/exif_orientation_editor.py (hunk 2 above)
Comment (heynemann) [reply thumbor/engines/extensions/exif_orientation_editor.py]: Not really. Fixed! 👍

File: thumbor/engines/extensions/exif_orientation_editor.py (hunk 3 above)
Comment (heynemann) [reply thumbor/engines/extensions/exif_orientation_editor.py]: Not really. Fixed! 👍

**Review by cristiandean — APPROVED** [review]

**Pull request merged**